machines/vic/config/config.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
```

Comment:
configuration is good. 
With this name, just because I thought go tends to use short name as their package, class name. So I used the short one here. :) 

For why I have a separate package here, the same reason with the previous comment. I think we need more clarification, or at least I need more clarification, for the design of metadata module.